features/add_warning_to_tools.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Warning About Budget Changes
+  In case we are unable to make all of the necessary changes to the redundancy advice tool ahead of Budget, we should have the following warning in place. This warning should be schedulable, and relatively straight-forward to put in place and remove. The same warning will need to be applied to other tools (see #7127) but also may need to be applied to other tools in case there are unforeseen Budget announcements that impact tools.
+  The following tool should have the warning if it has not been updated by March 16th. Redundancy Advice Tool - https://www.moneyadviceservice.org.uk/en/tools/redundancy-pay-calculator.
+
+  Scenario Outline: before the budget announcement
+    Given Today's date is "5th March 2017"
+    When I visit the Redundancy Advice Tool in "<language>"
+    Then I should not see the warning about the impending changes in "<language>"
+
+    Examples:
+      | language |
+      | english  |
```

Comment:
@thecountgs These should be capitalised (English, Welsh), I think these tests will fail otherwise?